torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -67,11 +67,65 @@ def list_with_default(out_size: List[int], defaults: List[int]):
 
 // Implementations of historic symbol behaviors are defined here
 // See note [Versioned Symbols]
+
+// This builtin is for testing
 auto _test_serialization_subcmul = R"SCRIPT(
 def _test_serialization_subcmul_0_2(self: Tensor, other:Tensor, alpha: number=2) -> Tensor:
   return other - (self * alpha)
 )SCRIPT";
 
+// Division versioned symbols, for Torchscript programs serialized when
+// division on integer tensors was floor division, not true division.
+// NOTE: updated division is not yet implemented. These versioned
```

Comment:
I'm not really sure how the last two sentences relate to each other.  Why can't you have (old) versioned symbols before the updated division is implemented?

Reply:
Ah, this NOTE should actually be removed. It was from when the PR was attempting to add the upgraders without actually throwing a runtime error. Unfortunately do to some complications those changes weren't separable.